spec/features/upload_download_mous_spec.rb

```diff
@@ -71,7 +71,8 @@
           it "lets them download the MoU" do
             visit @link
 
-            expect(page).to have_current_path(@link)
+            # as we're running locally, the actual asset lives under rails/active_storage/disk
+            expect(page.current_path).to start_with("/rails/active_storage/disk")
```

Comment:
Minor thing: I'd prefer the use of a variable instead of the hardcoded path, if one existed, but this is already a bit hacky as is anyway so I am not too bothered under the circumstances.

Reply:
I agree but since this probably won't be needed soon, I'm okay with this for now.